modules/vaos/app/services/vaos/configuration.rb

```diff
@@ -17,13 +17,18 @@ def service_name
     def connection
       Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
         conn.use :breakers
-
+        conn.request :camelcase
         conn.request :json
 
+        if ENV['VAOS_DEBUG'] && !Rails.env.production?
```

Comment:
We don't normally have debugging like this in modules. Is this a temporary thing that should have been removed or is it in for dealing with issues during UAT?

Is there a devops PR that goes with this ENV var?

Reply:
it's typically commented out... but with the amount of debugging necessary already for testing this, this is useful. You can see this used in Rx and SM configurations, but there it's commented out. ENV flag makes it much easier to run this on a staging machine.